modules/page_object_autofill_test_basic.py

```diff
@@ -60,7 +60,7 @@ def fill_input_element(self, ba: BrowserActions, field_name: str, term: str):
         ba.clear_and_fill(web_elem, term, press_enter=False)
 
     def click_form_button(self, field_name):
-        self.get_element("submit-button", field_name).click()
+        self.get_element("submit-button", labels=[field_name]).click()
```

Comment:
would prefer a type annotation on `field_name` like `field_name: str` or something!

Reply:
It is done